content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -172,24 +166,28 @@ For AMD GPUs, you can deploy [Node Labeller](https://github.com/RadeonOpenComput
   * AI - Arctic Islands
   * RV - Raven
 
+```shell
+kubectl describe node cluster-node-23
+```
+
 Example result:
+```
+Name:               cluster-node-23
+Roles:              <none>
+Labels:             beta.amd.com/gpu.cu-count.64=1
+                    beta.amd.com/gpu.device-id.6860=1
+                    beta.amd.com/gpu.family.AI=1
+                    beta.amd.com/gpu.simd-count.256=1
+                    beta.amd.com/gpu.vram.16G=1
+                    beta.kubernetes.io/arch=amd64
+                    beta.kubernetes.io/os=linux
+                    kubernetes.io/hostname=cluster-node-23
+Annotations:        kubeadm.alpha.kubernetes.io/cri-socket: /var/run/dockershim.sock
+                    node.alpha.kubernetes.io/ttl: 0
+…
+```
 
-    $ kubectl describe node cluster-node-23
-    Name:               cluster-node-23
-    Roles:              <none>
-    Labels:             beta.amd.com/gpu.cu-count.64=1
-                        beta.amd.com/gpu.device-id.6860=1
-                        beta.amd.com/gpu.family.AI=1
-                        beta.amd.com/gpu.simd-count.256=1
-                        beta.amd.com/gpu.vram.16G=1
-                        beta.kubernetes.io/arch=amd64
-                        beta.kubernetes.io/os=linux
-                        kubernetes.io/hostname=cluster-node-23
-    Annotations:        kubeadm.alpha.kubernetes.io/cri-socket: /var/run/dockershim.sock
-                        node.alpha.kubernetes.io/ttl: 0
-    ......
-
-Specify the GPU type in the pod spec:
+With the Node Labeller in use, you can specify the GPU type in the Pod spec:
```

Comment:
Node Labeller? What is it?

Reply:
Node Labeller is a 3rd party controller (mentioned earlier in the page). I've reworded with the aim of making this more clear.